src/utils/dateUtils.ts

```diff
@@ -1,20 +1,34 @@
 import { Dayjs } from 'dayjs';
 
-export const getNaiveDate = (dateString: string): Date => {
-    const date = new Date(dateString);
-    return new Date(
-        date.getUTCFullYear(),
-        date.getUTCMonth(),
-        date.getUTCDate(),
-        date.getUTCHours(),
-        date.getUTCMinutes(),
-    );
-};
+import { ZetkinEvent } from 'types/zetkin';
+
 
 export const getNewDateWithOffset = (date: Date, offset: number ):Date => {
     return new Date(new Date(date).setDate(date.getDate() + offset));
 };
 
+/**
+ * Returns the first and last object in an array of ZetkinEvents.
+ *
+ * This function works like this becuase the events will arrive from the
+ * server already sorted chronologically by start_time.
```

Comment:
It would be nice if this was not a requirement. Even though that should always be the case when events are coming directly from the server, I imagine cases where you might want to use this function for something like this:

```ts
const [firstEvent, lastEvent] = getFirstAndLastEvent([
  ...eventsFromCampaign1,
  ...eventsFromCampaign2,
]);
```

This would fail, and it would not be clear to the programmer that it will fail. The name of this function implies that it would work. So let's make it work, and add tests that verify it's working. 😊